kimchi/src/circuits/expr.rs

```diff
@@ -433,6 +433,25 @@ impl Op2 {
     }
 }
 
+/// The feature flags that can be used to enable or disable parts of constraints.
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(
+    feature = "ocaml_types",
+    derive(ocaml::IntoValue, ocaml::FromValue, ocaml_gen::Enum)
+)]
+pub enum FeatureFlag {
+    ChaCha,
+    RangeCheck,
+    ForeignFieldAdd,
+    Xor,
```

Comment:
every tag should be documented here :o

Reply:
Disagree, the variant names are descriptive enough.

Reply:
I don't think that bloating the code to appease a CI rule is a good way to go. It's supposed to work for us, not the other way around.